components/builder-api/src/server/mod.rs

```diff
@@ -65,7 +66,8 @@ features! {
         const Jobsrv = 0b0000_0010,
         const LegacyProject = 0b0000_0011,
         const Artifactory = 0b0000_0100,
-        const BuildDeps = 0b0000_1000
+        const BuildDeps = 0b0000_1000,
+        const EventBus = 0b0000_0110
```

Comment:
My understanding was features uses these as a bit mask. If that's the case wouldn't `EventBus == Jobsrv | Artifactory`?

Reply:
Well I might come across supremely ignorant here but: I'm not sure if features-rs is using these as a bitmask or not, I'll have to go read the macro. My understanding was that each entry in the features macro just needed a different integer expressed in binary notation. If these were equal then we would be seeing errors in the logs in live around artifactory failing to start/connect/etc.

Reply:
Having discussed a bit of this with @fnichol I think this should be fine. Also, I'm not running into any strange behavior in my functional tests when I toggle these options :)

Reply:
Yep, it's essentially a bitmask! The underlying implementation uses the [bitflags](https://docs.rs/bitflags) crate which tracks all these inside of a `usize` (I believe that's the packed primitive). So, lots and lots of potential features.